cmd/polywasm/index.html

```diff
@@ -9,13 +9,14 @@
 	<script>
 		// By default the ServiceWorker's scope will be "server/"
 		navigator.serviceWorker
-			.register('sw.js', { scope: "/app/" })
+			.register('sw.js', { scope: "/polyform/" })
```

Comment:
We need it to be a sub directory of the current directory for two reasons.

1. Local dev
2. Currently how it's set up, we will never be able to update the wasm unless it get's uncached. Reason being, everytime the user hits the page after the first install, the index.html hits the installed service worker, not the original HTML, which means it can't update any more.


I *think* this could work:

```suggestion
			.register('sw.js', { scope: window.location.pathname + "/app/" })
```